### ISSUE TITLE
Mullvad Browser crashes when watching Youtube

### ISSUE THREAD

**Issue by krlabs** [mullvad/mullvad-browser#188]
For the last 2 weeks, I have been observing a very strange phenomenon when Mullvad Browser crashes if Youtube tab is active and video is playing. For example, this happens when I watch a Youtube video in Mullvad and parallel working in another browser or application at the same time. But this happens randomly.

I cannot understand the nature of this phenomenon - I have not found any ideas or logs. I only know, that many years before this happen in the classic Mozilla Firefox for some bug. 

There is enough RAM, there is a reserve.  

Perhaps you, as the developers, can advise us on something, or at least tell us what to check. 

**Technical details:**

```
Name 	MullvadBrowser 13.0.4
Version 	115.5.0esr
Build ID 	20231116153520
Update Channel 	release
User Agent 	Mozilla/5.0 (X11; Linux x86_64; rv:115.0) Gecko/20100101 MullvadBrowser/115.5.0
OS 	Linux 5.10.0-kali3-amd64 #1 SMP Debian 5.10.13-1kali1 (2021-02-08)
Use recommended perfomance settings - Active
Mullvad VPN - Background Active
Mullvad Browser Extension - Active
```

**Comment by krlabs** [comment]
The problem could be caused by the uBlock Origin extension. We changed its settings. After the settings were rolled back to the default, the problem was not observed. But this is just a guess for now. We continue to monitor.

**Comment by krlabs** [comment]
> @krlabs Do you have an idea of what settings you used in uBO that created the crash?

 We have activated some security settings. Unfortunately, it's hard to say which ones right now. We will conduct an experiment and re-configure uBO. If this happens again, we will definitely send all information here.

**Comment by krlabs** [comment]
> @krlabs Do you have an idea of what settings you used in uBO that created the crash?

It's a shame to say, but as soon as the Mullvad Browser unexpectedly finished its work, it crashed and closed spontaneously. The tab with the video on Youtube was opened and the Chrome browser was opened in parallel. So it seems that the uBO plugin is no longer needed. Probably, it is something that occurs at the system - software or hardware levels. For me interesting, are there any logs where we can track the cause of the browser crash?

**Comment by krlabs** [comment]
We're closing this ticket because we haven't seen the problem repeatedly since then. Most likely, it was related to the graphics drivers on the Linux side. We reinstalled them and it hasn't happened again.
